### ISSUE TITLE
ccs-res cocosui is missing CloseNormal.png

### ISSUE THREAD

**Issue by stammen** [cocos2d/cocos2d-x#15803]
The ccs-res cocosui folder is missing CloseNormal.png. The following test fails on win32, win10 UWP and Windows 8.1 universal. 

9:Bugs Test #12 Crash after line 17:

```
    auto testSprite = Sprite::create("cocosui/CloseNormal.png");
```

cocosui/CloseNormal.png is missing. Please add it to ccs-res.

**Comment by stammen** [comment]
@minggo Both 9:Bugstests #12 and #13 are broken due to missing resources. They were broken by your commit https://github.com/dumganhar/ccs-res/commit/0c31b046de539d745ecd0ab522e6d45f97725c7d

Please remove the tests or replace the resources. Running Auto-test reveals these crashes.

**Comment by minggo** [comment]
@stammen Thanks, i will take a look.

**Comment by minggo** [comment]
fixed in #15804
